### PR TITLE
Feature/feature delete pet 2

### PR DESCRIPTION
**Pull request by danicm47**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePet.java  3
- app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestDeletePet.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5232056&bid=17152321)

File: app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestDeletePet.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import static org.junit.Assert.assertFalse;
+
+public class TestDeletePet {
+    private User user;
+    private Pet pet;
+    private TrDeletePet trDeletePet;
+
+    @Before
+    public void setUp() throws PetRepeatException {
+        user = new User("johnDoe", "albert69@gmail.com", "1234");
```
Comment (santidrj) [inline app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestDeletePet.java]: Codacy Issue found: [The String "albert69@gmail.com" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5232056)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/communication/InfoPetCommunication.java [hunk 2]
```diff
@@ -19,4 +20,6 @@
      * @param newImage The bitmap of the new image to display
      */
     void updatePetImage(Pet pet, Bitmap newImage);
+
+    void deletePet(Pet myPet) throws UserIsNotOwnerException;
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/communication/InfoPetCommunication.java:28]: You need to comment this method

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestDeletePet.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePet.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5232056&bid=17152321)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -72,8 +72,11 @@
     <string name="delete_pet_image_text">Are you sure you want to delete the profile image of your pet?</string>
     <string name="affirmative_response">Yes</string>
     <string name="negative_response">No</string>
+    <string name="delete_pet">DELETE PET</string>
```
Comment (AlbertPG) [inline app/src/main/res/values/strings.xml]: It is better if you put instead "Delete pet" and then add the next attribute to whenever you want it to appear all uppercase:
`android:textAllCaps="true"`
```suggestion
    <string name="delete_pet">Delete pet</string>
```

**Review by AlbertPG — COMMENTED** [review]
You should take a look at the comments I've written to you just before doing the merge.

**Review by AlbertPG — APPROVED** [review]
Okey, it looks right to me

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestDeletePet.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeletePet.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5232056&bid=17152321)

**Pull request merged**